### ISSUE TITLE
colorscheme seems bizzare when terminal use a light theme.

### ISSUE THREAD

**Issue by ch4m1gn0n** [LunarVim/LunarVim#4126]
### Problem description

1. terminal set a light theme. 
2. open lunarvim
3. dashboard seems normal, but file buffer's color looks bizarre.

seems like this one, #3932 , once i save the config file, the color scheme becomes successfully enabled.
i tried in wezterm and iterm2, both have this problem.

### LunarVim version

master-b1c72541

### Neovim version (>= 0.8.0)

NVIM v0.9.0 Build type: Release LuaJIT 2.1.0-beta3

### Terminal name

iTerm2, wezterm

### Operating system/version

macOS 13.3.1 22E261 arm64

### Screenshots

<img width="1427" alt="截屏2023-05-04 21 08 02" src="https://user-images.githubusercontent.com/51645204/236213869-146b4ea5-b0f8-4e08-8476-7e15cb138cb9.png">

**Comment by kobzar** [comment]
> This issue usually has something to do with the terminal itself. Can you check if the issue occurs in macOS default terminal emulator or iTerm2, for example?

The default osX Terminal.app isn't supported because dont support 256 colors. The topic starter check it in Iterm2 and wezterm. 
But the actual moment to @ch4m1gn0n did you run it without Tmux? If no - try without tmux.

**Comment by brneor** [comment]
Didn't notice tmux. In tmux this lines usually solve the issue:

```
set -g default-terminal "screen-256color"
set -ga terminal-overrides ",256col:Tc"
```

**Comment by ch4m1gn0n** [comment]
> Try to play with another themes https://www.lunarvim.org/docs/configuration/appearance/colorschemes#switching-colors
> 
> select one in config and restart the lvim.

the other themes work fine.

**Comment by ch4m1gn0n** [comment]
> This issue usually has something to do with the terminal itself. Can you check if the issue occurs in macOS default terminal emulator or iTerm2, for example?

iterm2 has the same problem. the default terminal theme will override all lvim theme.

**Comment by ch4m1gn0n** [comment]
> > This issue usually has something to do with the terminal itself. Can you check if the issue occurs in macOS default terminal emulator or iTerm2, for example?
> 
> The default osX Terminal.app isn't supported because dont support 256 colors. The topic starter check it in Iterm2 and wezterm. But the actual moment to @ch4m1gn0n did you run it without Tmux? If no - try without tmux.

I just ues wezterm,  without tmux.

**Comment by ch4m1gn0n** [comment]
I tried several wezterm built-in themes, it seems that all white themes have this problem.
```   -- color_scheme = "Ashes (light) (terminal.sexy)",
    -- color_scheme = "Hemisu Light (Gogh)",
    -- color_scheme = "3024 (light) (terminal.sexy)",
    -- color_scheme = "Google Light (Gogh)"
    color_scheme = "PaperColor Light (base16)"
    -- color_scheme = 'Harmonic16 Light (base16)'
    -- color_scheme = 'Default (light) (terminal.sexy)'
    -- color_scheme = "Atelier Heath Light (base16)",
```

**Comment by ch4m1gn0n** [comment]
Tried in lazyvim, also has same problem. but dark terminal theme and light nvim theme works fine.

**Comment by kobzar** [comment]
Try any light theme not base16

**Comment by ch4m1gn0n** [comment]
> 

Tried `Belafonte Day` ` AtomOneLight` `ayu_light`， all the same, do not work. Maybe I should make them have the same appearance mode.😭

**Comment by LostNeophyte** [comment]
the issue is probably caused by https://github.com/neovim/neovim/issues/22614

**Comment by LostNeophyte** [comment]
Is the problem fixed in nvim 0.9.1?
